frontend/pkg/frontend/middleware_logging_test.go

```diff
@@ -115,13 +121,15 @@ func TestMiddlewareLoggingPostMux(t *testing.T) {
 				w.WriteHeader(http.StatusOK)
 			}
 
-			MiddlewareLoggingPostMux(writer, req, next)
+			MiddlewareLogging(writer, req, func(w http.ResponseWriter, r *http.Request) {
+				MiddlewareLoggingPostMux(w, r, next)
+			})
```

Comment:
wrapped because that's what our router.go actually does and the first set of context is added much earlier now.